angular-oauth2-oidc/src/auth.config.ts

```diff
@@ -200,5 +200,12 @@ export class AuthConfig {
      * Normally, the discovey document's url starts with the url of the issuer.
      */
     public skipIssuerCheck? = false;
-    
+
+    /*
+     * final state sent to issuer is built as follows:
+     * state = nonce + nonceStateSeparator + additional state
+     * Default separator is ';' (encoded %3B).
+     * In rare cases, this character might be forbidden or inconvenient to use by the issuer so it can be customized.
+     */
+    public nonceStateSeparator = ';';    
```

Comment:
It should be optional to the user with a default value. Consider using `nonceStateSeparator?= ';'`

Reply:
Thx @alexndreazevedo, good point. It is changed.